tests/components/lock/test_verisure.py

```diff
@@ -0,0 +1,60 @@
+"""Tests for the Verisure platform."""
+
+from unittest.mock import Mock, patch
+from homeassistant.const import STATE_LOCKED, STATE_UNLOCKED
+
+
+NO_DEFAULT_LOCK_CODE_CONFIG = {
+    'locks': True,
+}
+
+DEFAULT_LOCK_CODE_CONFIG = {
+    'locks': True,
+    'default_lock_code': '9999',
+}
+
+
+async def test_verisure_no_default_code(hass):
+    """Test configs without a default lock code."""
+    with patch('homeassistant.components.lock.verisure.hub') as hub:
+        hub.config = NO_DEFAULT_LOCK_CODE_CONFIG
+
+        from homeassistant.components.lock.verisure import VerisureDoorlock
+        lock = VerisureDoorlock("lock")
```

Comment:
I suspected you'd ask me to do that, and commit https://github.com/home-assistant/home-assistant/pull/18873/commits/4a21d935985eb36b29795d28ef0f76e74fea5be0 has those changes.

It's a lot of setup and mocking to get the platform this way, especially for the two small unit tests that I wrote. I'm not sure that it's valuable in this particular instance to set up a whole platform since there are no tests for any of the other platform's behavior. Feels like wasted cpu cycles to me.